src/coreclr/src/vm/interoplibinterface.cpp

```diff
@@ -957,6 +955,9 @@ namespace InteropLibImports
                 externalObjectFlags,
                 gc.wrapperMaybeRef);
 
+            if (gc.objRef == NULL)
```

Comment:
>if (gc.objRef == NULL) [](start = 12, length = 22)

Boo... Tell me your thoughts on making the above have Try semantics? Not pushing for it, just curious.

Reply:
Ehhh. I thought about doing that too, but didn't really sway myself enough to make the change. You commenting on it as well indicates to me that I should do it.